Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -6,25 +6,21 @@
 
 public class NetworkObjectSpawner : NetworkBehaviour
 {
-    public NetworkObject prefabReference;
-    Random m_RandomGenerator = new Random();
+    [SerializeField] NetworkObject prefabReference;
 
-    public override void OnNetworkSpawn()
+    public void Start()
     {
-        base.OnNetworkSpawn();
-        if (!IsServer)
-        {
-            enabled = false;
-            return;
-        }
+        NetworkManager.Singleton.OnServerStarted += OnServerStartedIngredientSpawn;
+        enabled = false;
```

Comment:
What's the reason behind disabling the script? It no longer has an Update() , so methods will not be repeatedly executed anyway. We can get rid of this line.

Reply:
True! I did not realize that I am gonna remove that line.